src/libcollectionstest/vec_deque.rs

```diff
@@ -885,3 +885,11 @@ fn test_append() {
     assert_eq!(b.iter().cloned().collect::<Vec<_>>(), [1, 2, 3, 4, 5, 6]);
     assert_eq!(a.iter().cloned().collect::<Vec<_>>(), []);
 }
+
+#[test]
+fn test_retain() {
+    let mut buf = VecDeque::new();
+    buf.extend(1..4);
```

Comment:
An unrelated test was failing causing the tests to abort early. However, the abort was lost well past my scrollback buffer and the old test logs were still there so I figured that my test passed...